src/main/java/cofh/thermaldynamics/duct/light/LightGrid.java

```diff
@@ -114,7 +115,7 @@ public void setLight(boolean lit) {
 				for (ChunkCoord chunk : chunks) {
 
 					PlayerChunkMapEntry inst = manger.getEntry(chunk.chunkX, chunk.chunkZ);
-					if (inst != null && inst.players.contains(player)) {
+					if (inst != null && inst.containsPlayer((EntityPlayerMP) player)) {
```

Comment:
Same deal, this is likely from 1.7 and didn't have such a method. But either way, the AT doesn't really cost anything, and the cast to EntityPlayerMP does.